tests/win_sec_attr.c

```diff
@@ -102,44 +102,36 @@ int main ()
     test_send (sc, "0123456789012345678901234567890123456789");
     test_recv (sb, "0123456789012345678901234567890123456789");
 
-    ret3 = nn_getsockopt(sb, NN_IPC, NN_IPC_SEC_ATTR, &void_ret_value, &void_ret_size);
-    nn_assert(ret3 == 0);
-    nn_assert(void_ret_value == &sec);
-
+    ret3 = nn_getsockopt (sb, NN_IPC, NN_IPC_SEC_ATTR, &void_ret_value, &void_ret_size);
+    nn_assert (ret3 == 0);
+    nn_assert (void_ret_value == &sec);
 
-    // verify that the pipe has the same security descriptor that we set by comparing the ace of the kernel object
-    // to the one we created it with
-    pipeHandle = CreateFileA(
-        "\\\\.\\\\pipe\\" PIPE_NAME,
-        READ_CONTROL,
-        0,
-        NULL,
-        OPEN_EXISTING,
-        FILE_FLAG_OVERLAPPED,
-        NULL
-        );
 
-    nn_assert(pipeHandle != INVALID_HANDLE_VALUE);
+    /*  Verify that the pipe has the same security descriptor that we set by
+        comparing the ace of the kernel object to the one we created it with. */
+    pipeHandle = CreateFileA ("\\\\.\\\\pipe\\" PIPE_NAME, READ_CONTROL, 0, NULL, 
+        OPEN_EXISTING, FILE_FLAG_OVERLAPPED, NULL);
 
+    nn_assert (pipeHandle != INVALID_HANDLE_VALUE);
 
-    ret2 = GetSecurityInfo(pipeHandle, SE_KERNEL_OBJECT, DACL_SECURITY_INFORMATION, NULL, NULL, &dacl, NULL, &sd);
+    ret2 = GetSecurityInfo (pipeHandle, SE_KERNEL_OBJECT, DACL_SECURITY_INFORMATION,
+        NULL, NULL, &dacl, NULL, &sd);
 
-    nn_assert(ret2 == ERROR_SUCCESS);
-    nn_assert(1 == dacl->AceCount);
+    nn_assert (ret2 == ERROR_SUCCESS);
+    nn_assert (1 == dacl->AceCount);
 
-    bret = GetAce(dacl, 0, &ace);
+    bret = GetAce (dacl, 0, &ace);
 
-    nn_assert(bret == TRUE);
-    nn_assert(ace->AceType == ACCESS_ALLOWED_ACE_TYPE);
+    nn_assert (bret == TRUE);
+    nn_assert (ace->AceType == ACCESS_ALLOWED_ACE_TYPE);
 
-    allowed_ace = (PACCESS_ALLOWED_ACE)ace;
-    the_sid = (PSID)&(allowed_ace->SidStart);
+    allowed_ace = (PACCESS_ALLOWED_ACE) ace;
+    the_sid = (PSID) &(allowed_ace->SidStart);
 
-    nn_assert(IsValidSid(the_sid));
+    nn_assert (IsValidSid (the_sid));
 
-    equal = EqualSid((PSID)&(allowed_ace->SidStart), &SIDAuthUsers);
-    nn_assert(equal);
-    LocalFree(dacl);
```

Comment:
Yes; and @Snaipe, good call; I'm going to audit and see how many `LocalFree` need to be called ...